server/src/metadata.js

```diff
@@ -231,8 +231,9 @@ class Metadata {
       // Redirect the 'users' table to the one in the internal db
       this._collections.set('users', new Collection({ id: 'users', table: 'users' },
                                                     this._internal_db, this._conn));
-    }).catch(() => {
+    }).catch((e) => {
       this.close();
+      throw e;
```

Comment:
I'll upload a new PR in a minute - there were a couple other wonky things I noticed.